src/main/query_profiler.cpp

```diff
@@ -333,7 +342,7 @@ static void ToJSONRecursive(QueryProfiler::TreeNode &node, std::ostream &ss, int
 	ss << "{\n";
 	ss << string(depth * 3, ' ') << "\"name\": \"" + node.name + "\",\n";
 	ss << string(depth * 3, ' ') << "\"timing\":" + StringUtil::Format("%.2f", node.info.time) + ",\n";
-	ss << string(depth * 3, ' ') << "\"cardinality\":" + to_string(node.info.elements) + ",\n";
+	ss << string(depth * 3, ' ') << "\"cardinality\":" + std::to_string(node.info.elements) + ",\n";
```

Comment:
Please include `duckdb/common/to_string.hpp` rather than changing this to std::to_string